lib/commands.py

```diff
@@ -277,7 +277,7 @@ def add_group(document, node, command):
         node.getparent(),
         SVG_GROUP_TAG,
         {
-            "id": generate_unique_id(document, "group"),
+            "id": generate_unique_id(document, "command_group"),
```

Comment:
The problem with using this as a way to detect commands is that it won't work on existing SVGs that people made with earlier versions. :(

Reply:
I supposed it doesn't really have to. This is because we don't expect clones in former designs. It is only used if we are inside a clone element. I couldn't find any usage of the command group id somewhere else.

Reply:
Actually, cloning an entire design is exactly what some folks have requested (I think).  They want to be able to sew out multiple copies of a design, and make layouts with multiple copies.  They want to be able to edit the original and have the changes reflected in the copies.

Fortunately, I think we can continue to detect clones in the same way we have in the past, so we oughtn't need to depend on the ids being named "command_group*".

All that said, naming the IDs this way makes sense anyway, just from a tidiness perspective :)